src/fmu/pem/pem_utilities/enum_defs.py

```diff
@@ -52,3 +52,19 @@ class RPMType(str, Enum):
 class VolumeFractions(str, Enum):
     NTG_SIM = "ntg_sim"
     VOL_FRAC = "fraction_files"
+
+
+class GasModels(str, Enum):
+    GLOBAL = "Global"
+    LIGHT = "Light"
+    HC2016 = "HC2016"
```

Comment:
Wondering why `HC2016` has a different color... it is not a reserved keywords in Python as far as I know.